### ISSUE TITLE
Importing a list type from another file results in the loss of the list values type

### ISSUE THREAD

**Issue by iansan5653** [microsoft/pyright#284]
**Describe the bug**
If I create a variable that holds a `List` type (with the values type defined) in one file and import it into another file, the type is imported as `List[unknown]`.

**To Reproduce**
Create a variable in one file with a defined list type, ie `List[int]`. Import it into another file. In the original file the variable will be treated properly. In the other file, it will be `List[unknown]`.

**Expected behavior**
The type should be preserved everywhere.

**Screenshots or Code**
file_a.py:
```py
import typing

Example = typing.List[int]

Example # Has type `Type[List[int]]`
```

file_b.py:
```py
import file_a

file_a.Example # Has type `Type[List[unknown]]`
```

**VS Code extension or command-line**
I'm running pyright as an extension, 1.0.70.

**Comment by erictraut** [comment]
Thanks for the bug report. I'm able to repro the problem, and I'll work on a fix.

**Comment by erictraut** [comment]
Found it. The fix will be in the next release.

**Comment by erictraut** [comment]
This is fixed in 1.0.71, which I just published.
